ansible/install-apps.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
+  tasks:
+  - stat:
+      path: {{ apt_repo_list_file }}
```

Comment:
I've integrated a linter for you in this PR: https://github.com/kpi-web-guild/devspace/pull/6/checks?check_run_id=104751907